dlt/extract/incremental/__init__.py

```diff
@@ -111,6 +112,8 @@ class Incremental(ItemTransform[TDataItem], BaseConfiguration, Generic[TCursorVa
     row_order: Optional[TSortOrder] = None
     allow_external_schedulers: bool = False
     on_cursor_value_missing: OnCursorValueMissing = "raise"
+    range_start: TIncrementalRange = "closed"
```

Comment:
do not forget docstrings and docs

Reply:
also add this to `TIncrementalArgs` (so we can define those in REST API)

Reply:
also when start_range is open - disable boundary deduplication. there's no reason to deduplicate. there's no boundary overlap

Reply:
Added all here